doc/setup.md

```diff
@@ -0,0 +1,287 @@
+
+- the public key `ephemeral_public_key[i]` which indicates the signer who sends this vss.
+- the public key `ephemeral_public_key[j]` which indicates the signer to be received the `ephemeral_vss[i, j]`
+- "positive"[^6] public commitments `commitments[k]` (k = 1, 2, ..., t)
+- "positive" secret `secret[j]` to perform secret sharing scheme.
+- "negative" public commitments `commitments[k]` (k = 1, 2, ..., t)
+- "negative" secret `secret[j]` to perform secret sharing scheme.
+
+`ephemeral_public_key[i, j]` is encrypted in the same way as `static_vss` described in Step 3.
+
+Signer[i] sends the generated values `ephemeral_vss[i, j]` to other signers.
+
+[^6]: In schnorr signature schema used in Tapyrus Core, we use the random value "r" and the ephemeral point "R" on the elliptic curve, where R = rG (G is the generator of the curve). In Tapyrus signature schema we should choose R so that jacobi(y(R)) = 1 (if not, use (n - r) instead of r and generate R so that R = (n - r)G, where n is the order of the curve). So in this step, we should generate both "positive" secret and commitments that corresponds to r, and "negative" ones that corrensponds to (n - r). For more information, see [Tapyrus Schnorr Signature Specification](https://github.com/chaintope/tapyrus-core/blob/master/doc/tapyrus/schnorr_signature.md)
+
+## Step 8. Sign the genesis block locally.
+
+Signer[i] secretly receives `ephemeral_vss[j, i]` generated by other signers. Then Signer[i] generates `local_sig` using `tapyrus-setup sign`.
+To generate `local_sig`, Signer[i] need at least t valid `ephemeral_vss[j, i]`s. In the following steps, we suppose that Signer[1], Signer[2], ... Signer[t] are honest signers providing a valid vss and Signer[i] got `ephemeral_vss[j, i]`(j = 1, 2, 3, ..., t).
+
+```
+tapyrus-setup sign -static_vss=1:<static_vss[1, i]> -static_vss=2:<static_vss[2, i]> ... -static_vss=t:<static_vss[t, i]> -ephemeral_vss=1:<ephemeral_vss[1, i]> -ephemeral_vss=2:<ephemeral_vss[2, i]> ... -ephemeral_vss=t:<ephemeral_vss[t, i]>   -aggregated_public_key=<aggregated_public_key> -nodesecret=<node_secret[i]> -private_key=<static_private_key[i]> -block=<block>
+
+output: <local_sig[i]>
+```
+
+- `static_vss[j, i]` (j = 1, 2, ..., t) are the static VSSs generated by Step 3.
+- `ephemeral_vss[j, i]` (j = 1, 2, ..., t) are the ephemeral VSSs generated by Step 7.
+- `static_private_key[i]` is the static private key of Signer[i], generated in Step 2.
+- `aggregated_public_key` is an aggregated public key, generated in Step 4.
+- `node_secret[i]` is the secret key share of Signer[i], generated in Step 4.
+- `block` is the genesis block without block proof.
+- `local_sig[i]` is the "local signature" described as γi in the paper [^4]. signer[i] may reveal `local_sig[i]`.
+
+Then each signer broadcasts their local_sig.
+
+## Step 9. Compute the signature of the genesis block.
+
+After collecting `local_sig`, we can compute the signature of the genesis block using `tapyrus-setup combinesig`
```

Comment:
I think `combine` is not correct. A final signature is computed from local sigs but it is not combined.

How about just `computesig`?

Reply:
Fixed at 2b336fb14ac15b0023041d4f25e7e109bdcaa341